### ISSUE TITLE
Choosing a model

### ISSUE THREAD

**Issue by mcgregorian1** [SCBI-ForestGEO/McGregor_climate-sensitivity-variation#9]
Hi @teixeirak 

I've spent the morning reading up on AIC and R^2 values and I have some updates before I move forward with #7 

1. First of all, I was running the models with NA values. Removing them, however, doesn't change the model ranking.

2. "year" should be fixed effect, but I already mentioned this and it doesn't really affect the model ranking.

3. I was reading a number of posts saying how AIC is great for comparing similar models, but ultimately the "best" model based on AICc values isn't necessarily the "best" model overall. They suggested using R^2 values to give a comparison. Doing this for the models gives me a range of 0.084 - 0.095. In other words, in reality, none of the models explain more than 10% of any variation in the resistance values.

Here's what an R^2 value would look like that we were hoping to see:
![image](https://user-images.githubusercontent.com/39522449/54701511-85060b00-4b0b-11e9-866d-19ebbf357c6b.png)

4. I decided to check plots of this to be sure and, it turns out there are indeed almost no trend lines. dbh_old gives the closest to an effect, but even then the relationship is slight.
![image](https://user-images.githubusercontent.com/39522449/54701072-9995d380-4b0a-11e9-85da-f9b1c16c478d.png)

![image](https://user-images.githubusercontent.com/39522449/54701141-c944db80-4b0a-11e9-8728-5c1b22ce09ff.png)

![image](https://user-images.githubusercontent.com/39522449/54701033-82ef7c80-4b0a-11e9-8ade-0121755f10e7.png)

![image](https://user-images.githubusercontent.com/39522449/54701111-b4684800-4b0a-11e9-9b60-e42101d79dbc.png)

5. Here is another visualization showing the predicted values of dbh_old (the line) based on the regression equation plotted with the actual observations. The trend is there but very very slight.

![image](https://user-images.githubusercontent.com/39522449/54701383-4bcd9b00-4b0b-11e9-8591-3caed8b9558b.png)

## Next steps
I'm not sure where to go from here. Valentine quickly was trying to maybe fit this to a generalized linear mixed model as opposed to a linear mixed model, but at this point I'm nervous about grasping for straws given already how I've failed to flag this lack of fit from earlier.

In short, it seems I've been putting too much faith in AICc without fully understanding the backstory behind what it means, even though I thought I knew.
